lib/rules/media-feature-range-notation/__tests__/index.mjs

```diff
@@ -233,3 +313,44 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: ['prefix', { except: ['exact-value'] }],
+
```

Comment:
```suggestion
	fix: true,
	computeEditInfo: true,
```

I believe the `@media (width: 1px)` test should be fixable with the existing `context` fixer.

Reply:
Done